src/main/java/org/broadinstitute/consent/http/service/DarCollectionService.java

```diff
@@ -664,19 +678,25 @@ public DarCollection cancelDarCollectionElectionsAsChair(DarCollection collectio
   public DarCollection createElectionsForDarCollection(User user, DarCollection collection)
       throws Exception {
     try {
-      List<String> createdElectionReferenceIds = collectionServiceDAO.createElectionsForDarCollection(
-          user, collection);
+      DataAccessRequest dar = collection.getMostRecentDar();
+      List<String> createdElectionReferenceIds = collectionServiceDAO.createElectionsForDarByUser(
+          user, dar);
       if (createdElectionReferenceIds.isEmpty()) {
         var e = new IllegalStateException(
-            "No elections were created for DAR Collection: %s".formatted(
-                collection.getDarCode()));
+            "No elections were created for DAR Collection: %s %s".formatted(
```

Comment:
The collection is ambiguous.  Adding the reference ID makes it clearer about what was happening at the time this error was encountered.